config.test.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
If it's .json then Github probably expects all keys and string values to be surrounded in double quotes